src/source/JavaGenerator.scala

```diff
@@ -89,6 +92,7 @@ class JavaGenerator(spec: Spec) extends Generator(spec) {
     for (c <- consts) {
       writeDoc(w, c.doc)
       javaAnnotationHeader.foreach(w.wl)
+      marshal.nullityAnnotation(c.ty).foreach(w.wl)
```

Comment:
Hm, I don't think this makes sense - this annotation is on the class itself, where neither nullable or checkfornull would make sense, right?